lib/engine/game/g_1880/step/assign.rb

```diff
@@ -9,13 +9,11 @@ module Step
         class Assign < Engine::Step::Assign
           ACTIONS_WITH_PASS = %w[assign pass].freeze
           def actions(entity)
-            return [] if entity.player?
+            return [] if entity.player? || current_entity.minor?
```

Comment:
these changes are a no-op with the intent to make the checks a bit more readable

p5_block does the same checks, so the functionality is the same

will be slightly slower since we descend into p5_block instead of short-circuiting earlier

Reply:
looks good, thanks for putting it all in p5_block, instead of the existing p5_block || condition.

Very nitpicky, and feel free to ignore, but maybe it should be p5_block? instead of p5_block

Reply:
not sure I understand, the current patch has p5_block? as the function name, if that's what you're describing